core/templates/dev/head/directives.js

```diff
@@ -156,3 +156,17 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('updateLanguage', ['$rootScope', function($rootScope) {
```

Comment:
Hm, not sure I understand. If you already have Angular two-way binding in the controller, why would that require a separate function to update the lang attribute? Don't you just set $scope.currentLang to whatever it should be?